src/api/app/views/layouts/webui/_announcement.html.haml

```diff
@@ -0,0 +1,5 @@
+- announcement = User.current.pending_announcement
+- if announcement.present?
+  .container_16#announcements
+    .ui-widget.grid_16.alpha.omega.announcement
+      = announcement_content(announcement)
```

Comment:
Why did you create a helper for this? 🤔

Reply:
> Why did you create a helper for this?

Go to the beach :beach_umbrella:, you have public holiday :smile:

Reply:
Btw: I also don't like the helper too much ...

Reply:
That's a leftover from when we had 2 places to render the ToS. We can remove it now